Presentation.Web/Controllers/API/V1/UserController.cs

```diff
@@ -224,6 +251,16 @@ private static UserWithOrganizationDTO ToUserWithOrgDTO(UserRoleAssociationDTO d
             return new(dto.User.Id, dto.User.GetFullName(), dto.User.Email, dto.Organization.Name, dto.User.HasApiAccess.GetValueOrDefault(false));
         }
 
+        private static IEnumerable<UserWithEmailDTO> ToUserWithEmailDtos(List<User> users)
+        {
+            return users.Select(ToUserWithEmailDto).ToList();
+        }
+
+        private static UserWithEmailDTO ToUserWithEmailDto(User user)
+        {
+            return new(user.Id, user.Name, user.Email);
```

Comment:
check if there isn't already an extension method for this... then the "fullname" issue will not be an issue to you

Reply:
I couldn't find any extension method for this mapping, so I created one in the DTOMappingExtensions

Reply:
Great: Please reuse it here: 
![image](https://user-images.githubusercontent.com/13116889/170995007-c4192c86-7ef4-4839-a758-82b61039b4bb.png)